WordPress/src/main/java/org/wordpress/android/ui/posts/services/PostUploadService.java

```diff
@@ -1006,13 +994,16 @@ public void updateNotificationProgress(float progress) {
             }
 
             mNotificationBuilder.setProgress(100, (int)Math.ceil(currentChunkProgress), false);
+            notifyManager(mNotificationId, mNotificationBuilder.build());
+        }
 
+        private void notifyManager(int id, Notification notification) {
```

Comment:
Nitpicking but, for readability purposes, would you consider renaming this to something like "doNotify()" to better reflect its functionality? Cheers!